src/library_wasmfs.js

```diff
@@ -107,7 +107,11 @@ mergeInto(LibraryManager.library, {
       // TODO: Remove dependency on FS.cwd().
       // User code should not be using FS.cwd().
       // For file preloading, cwd should be '/' to begin with.
-      return '/';
+      return withStackSave(() => {
+        var buffer = stackAlloc({{{PATH_MAX}}});
```

Comment:
`PATH_MAX` is defined in C code, so this will not just work as it is. Instead, `{{{ cDefine('PATH_MAX') }}}` is what we want - that looks up a C define. Note that you must also add `PATH_MAX` to `src/struct_info.json`.